.github/workflows/app-publish.yaml

```
@@ -0,0 +1,121 @@
+name: App Publish
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release*
```

```suggestion
      - master
```

No need to run on release branches because we (normally) only push when we're going to tag it.